warcry_proj/README.txt

```diff
@@ -0,0 +1,22 @@
+warcry_proj README
+==================
+
+Getting Started
+---------------
+
+- Login to Vagrant
+* vagrant up
+* vagrant ssh
+
+- Ensure the bootstrap script is run to configure and install all required packages
+* sudo script/bootstrap vagrant
```

Comment:
We may want to consider adding another playbook for this... adding a "project.yml" so that you would run /script/bootstrap project

We could have the system run bootstrap project as part of the provisioning if we want as well... something we should probably discuss as to what might be the best option. I'm not sure if we should be adding the installation of project specific items in the vagrant playbook or not... *ponders*